src/io/flutter/run/daemon/DeviceDaemon.java

```diff
@@ -218,11 +218,6 @@ else if (process.isProcessTerminated()) {
           try {
             ready.get(100, TimeUnit.MILLISECONDS);
 
-            // Try not to show a partial device list.
-            // It currently takes 4+ seconds for the devices to show up.
-            // Remove when fixed: https://github.com/flutter/flutter/issues/8439
```

Comment:
Not yet, but I have a local change in progress to fix it, and this delay is not actually necessary IMHO. We only show one device at a time, so trying not to display a partial list doesn't make a lot of sense.